### DISCUSSION TITLE
关于get_image里的问题

### DISCUSSION THREAD

**Discussion by hakaboom** [Meituan-Dianping/vision-ui#22] [Q&A] [answered]
https://github.com/Meituan-Dianping/vision-ui/blob/1d218217d7c31e8258184affba5b707fa353ca23/service/image_similar.py#L24
这里的0.078的意义是什么

**Comment by BrightHai** [comment, marked as answer]
OpenCV中图像的表示为image[h-高, w-宽, c-通道]，对于移动端UI，h方向起点区域存在信号状态栏，不同屏幕大小的设备可以用 w*缩放系数表示那部分区域
